compat/hwc2/hwc2_compatibility_layer.cpp

```diff
@@ -214,7 +214,11 @@ hwc2_error_t hwc2_compat_display_present(hwc2_compat_display_t* display,
     android::sp<android::Fence> presentFence;
     HWC2::Error error = display->self->present(&presentFence);
 
-    *outPresentFence = presentFence->dup();
+    if (presentFence) {
```

Comment:
On my build environment, i need change to `if (presentFence != NULL) {`.
Tested on vince, no lipstick crashes.

Reply:
> On my build environment, i need change to `if (presentFence != NULL) {`.
> Tested on vince, no lipstick crashes.

Does that also work for you @NotKit? if so maybe we can change it so it builds everywhere and merge this.

Reply:
Tested to work for me. I modified the commit, let's merge this then.